src/Tribe/Admin/Activation_Page.php

```diff
@@ -212,7 +215,24 @@ public function showed_update_message_for_current_version() {
 		if ( empty( $message_version_displayed ) ) {
 			return false;
 		}
-		if ( version_compare( $message_version_displayed, $this->args['version'], '<' ) ) {
+
+		$current_version = explode( '.', $this->args['version'] );
+		if ( count( $current_version ) > 2 ) {
+			$current_version = array_splice( $current_version, 0, 2 );
+		} elseif ( count( $current_version ) < 2 ) {
+			$current_version[] = 0;
```

Comment:
This only adds one "digit", perhaps: 
```suggestion
			$current_version[] = array_pad( $current_version, 3, '0' );
```

Reply:
that 3 could be 4 - then everything gets padded to a "hotfix version"